10/umbraco-forms/developer/extending/customize-default-workflows.md

```diff
@@ -254,4 +254,27 @@ Again, you will need to register your custom class, for example, in a composer w
 
 ```csharp
 builder.Services.AddUnique<IApplyDefaultFieldsBehavior, CustomApplyDefaultFieldsBehavior>();
-```
\ No newline at end of file
+```
+
+### Example - Excluding a Built-in Field
+
+The following class shows how to exclude built-in field types using a custom composer. The `Password`, `Recaptcha2` and `RichText` field types (or "answers") will no longer be available for selection when creating a form in the backoffice.
+
+```
+using Umbraco.Forms.Core.Providers;
+using Umbraco.Forms.Core.Providers.FieldTypes;
+
+namespace MyNamespace
+{
+    public class MyFormFieldsComposer : IComposer
+    {
+        public void Compose(IUmbracoBuilder builder)
+        {
+            builder.WithCollectionBuilder<FieldCollectionBuilder>()
+               .Exclude<Password>()
+                .Exclude<Recaptcha2>()
+                .Exclude<RichText>();
```

Comment:
```suggestion
            builder.FormsFields()()
                .Exclude<Password>()
                .Exclude<Recaptcha2>()
                .Exclude<RichText>();
```

Reply:
This syntax improvement is great to know - I will use this approach in my code!

I was struggling to find how to do this, so it is going to be very useful. I am not sure of support for v11/12 (since we only use LTS) so perhaps someone could confirm this for v11/12 and move this content to wherever you think is most appropriate?

Reply:
Yes, this would be the same across 11 and 12.  Over to you I think @alina-tincas to see if you can migrate this to an appropriate place (you can consider my suggestions above).

Reply:
@AndyButland do you think this example can be added here: https://docs.umbraco.com/umbraco-forms/developer/extending/adding-a-fieldtype as this is about field types? Because adding a new article we need to have a bit more content 🤔

Reply:
It could, but actually I would suggest that even though the page would be a bit light on content, it's probably better as a new page.  The reason being that removing existing fields and adding new fields, whilst very related in a technical sense, are fairly different when it comes to the use case of why you'd want to do one or other of these things.

In the former case, it's because for example you have no use for file upload and don't want editors using them.

In the latter, it's because there's some new functionality you need you want to build yourself.

I could see people looking to do either of these things, but not generally both at the same time.

Reply:
Thank you for confirming Andy! I have created a new PR for this here: #5486